packages/Chem/src/widgets/scaffold-tree.ts

```diff
@@ -349,11 +348,21 @@ export class ScaffoldTreeViewer extends DG.JsViewer {
   Table: string;
   treeEncode: string;
 
+  sizesMap: {[key: string]: {[key: string]: number}} = {
```

Comment:
This is not typed well enough. Clearly you rely on instances having "height" and "width", this should be reflected.

Reply:
Fixed